README.md

```diff
@@ -48,131 +44,6 @@ bundle install
-
-If you don't have an account ask our local administrator to create one for you.
-
-1. You will get an invitation email and be prompted to create an account
-1. Once the account is created, wait for a 2nd email to activate that account
-1. Once you've activated the account, go back to the first email and use the
-   bottom link to accept the invitation. You should now have access to our keys.
-
-### Creating Alma Fixtures
-
-In the API sandbox (see above)
-
-1. Select the 'api-na' north america server
-1. Select the read-only API key
-1. Click the api endpoint you want to use
-1. Click 'try it out'
-1. Set all desired parameters
-1. Select media type: application/json or application/xml (below the 'Execute'
-   button)
-1. Click 'Execute'
-1. You can download the file with the little "Download" button
-
-### Export a set of test records from production
-1. Login to alma https://princeton.alma.exlibrisgroup.com/SAML
-1. In the left side bar click 'Admin' -> Select 'Manage sets'
-1. Find or create the set you want to use.
-1. Click on the elipsis button of the set. -> Select 'Members'
-1. If there are records in the set that it is not desired to export, select the records using the checkbox to the left and click 'Remove Selected'
-1. Click 'Add Members'. Add in the search bar the desired mms_id. -> 'Search' -> Select the listed record using the checkbox -> Click Add Selected.
-1. In the left bar, click 'Resources' -> 'Publishing Profiles' -> Find the 'DRDS Test Record export' publishing profile.
-1. -> Click the elipsis button and select 'Edit'. Configure it to use your set under "Content". Click "Save".
-1. -> Click the elipsis button and select 'Republish'. -> Select 'Rebuild Entire Index' -> Click 'Run Now'.
-1. The new tar.gz file with the selected records will be on the lib-sftp server as '/alma/drds_test_records_new[_i].tar.gz'
-
-### Finding a Voyager item in Alma
-
-Voyager items, once the migration is finished, will have an ID in Alma equal to
-`99<voyager_id>3506421`
-
-### Hitting the Alma API
-
-The Alma web API has a maximum concurrent hit limit of 25 / second. The API limits are documented at https://developers.exlibrisgroup.com/alma/apis/#threshold and Daily use stats can be viewed at https://developers.exlibrisgroup.com/manage/reports/.
-
-## ARK Caching
-
-In order to resolve bibliographic identifiers (bib. IDs) to resources with ARKs and IIIF manifests for resources managed within digital repositories, caches are seeded and used in order to resolve the relationships between these resources.
-
-### Seeding the Cache
-
-One may seed the cache using the following Rake Task:
-```bash
-rake liberate:arks:seed_cache
-```
-
-In development, when running commands that utilize the cache, such as commands indexing via traject, set the `FIGGY_ARK_CACHE_PATH` to point to `spec/fixtures/marc_to_solr/figgy_ark_cache` in the local environment.
-```bash
-export FIGGY_ARK_CACHE_PATH=spec/fixtures/marc_to_solr/figgy_ark_cache
-```
```

Comment:
It would be nice to keep these steps in the README, since they can trip up setting a dev environment.

Reply:
Moved back to README